Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -358,7 +358,7 @@ public void DeleteItem(BaseItem item, DeleteOptions options, BaseItem parent, bo
 
             var children = item.IsFolder
                 ? ((Folder)item).GetRecursiveChildren(false)
-                : Enumerable.Empty<BaseItem>();
+                : Array.Empty<BaseItem>();
```

Comment:
Here I just suppress false multiple enumeration warning. So now `children` is an `IList`